rules/catch-error-name.js

```diff
@@ -46,8 +46,8 @@ const create = context => {
 		ignore: [],
 		...context.options[0]
 	};
-	const {name: expectedName} = options;
-	const ignore = options.ignore.map(
+	const {name: expectedName, ignore: ignore_} = options;
```

Comment:
I don't think it should enforce destructuring if it conflicts with an existing identifier, like here.